IntegrationTests/tests_04_performance/test_01_resources/test_1000_tcpconnections.swift

```diff
@@ -37,13 +43,12 @@ func run(identifier: String) {
         for _ in 0 ..< numberOfIterations {
             let clientChannel = try! clientBootstrap.connect(to: serverChannel.localAddress!)
                     .wait()
-            defer {
-                try! clientChannel.close().wait()
-            }
             // Send a byte to make sure everything is really open.
             var buffer = clientChannel.allocator.buffer(capacity: 1)
             buffer.writeInteger(1, as: UInt8.self)
-            try! clientChannel.writeAndFlush(NIOAny(buffer)).wait()
+            _ = clientChannel.writeAndFlush(NIOAny(buffer))
```

Comment:
Same here; `promise: nil` instead of dropping the future. Although this should drop our allocation count.